### PR TITLE
Reduce code duplication

### PR DESCRIPTION
**Pull request by HoLLy-HaCKeR**
This PR replaces some repeated command handlers to use a single abstraction method instead of duplicating a lot of code. It also updates the gitignore file and fixes up some link commands.

### CODE REVIEW

**Comment by HoLLy-HaCKeR** [comment]
Linter errors don't seem to be related to any code changes in this PR.

File: main.go [hunk 1]
```diff
@@ -95,6 +95,18 @@ func registerCommand(c command.Command) {
 	helpData[c.Category][c.Name] = c
 }
 
+func registerHardcodedCommand(name, helpText, category, response string) {
```
Comment (meyskens) [inline main.go:98]: Can we move this logic into `registerLinkCommand` as it only is used there?

**Comment by meyskens** [comment]
Thank you very much for this PR! Just one tiny nit :)

File: main.go (hunk 1 above)
Comment (HoLLy-HaCKeR) [reply main.go]: Done.

**Pull request merged**